### PR TITLE
feat: added dashboard and quotes feature

### PR DESCRIPTION
**Pull request by zachmolony**

### CODE REVIEW

**Comment by Kunle2021** [comment]
After discussion we changed the dashboard to show nearby location and skillset of the tradesman

**Comment by Kunle2021** [comment]
Discussions occurred in a google meet call

File: src/context/quotesContext.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			cost: 50,
+			date: '15/4/2022',
+		},
+	])
+	const [loading, setLoading] = useState(false)
+	const [error, setError] = useState('')
+
+	const createQuote = values => {
+		setLoading(true)
+		// axios.post('', values)
```
Comment (Kunle2021) [inline src/context/quotesContext.js:25]: Need to Create a call to the quote table in the firebase database - send quote data

File: src/Components/Form/SignIn.js [hunk 2]
```diff
@@ -1,37 +1,42 @@
-import React, { useRef } from "react";
```
Comment (Kunle2021) [inline src/Components/Form/SignIn.js:1]: This file is causing an error in the build

Comment (zachmolony) [reply src/Components/Form/SignIn.js:1]: should be fixed now

File: src/context/quotesContext.js (hunk 1 above)
Comment (zachmolony) [reply src/context/quotesContext.js:25]: yeah defo - let's add this in another branch after we merge

**Pull request merged**